TomeOfKnowledge.playground/Pages/Exploring Expressions.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,50 @@
+//: ## Exploring Expressions
+//: 
+//: Our expressions are more robust than what is needed for typical tabletop play. In fact, our generic expression system is capable of modeling expressions that would otherwise be considered too cumbersome or complex to physically roll. In order to help developers understand what their expressions actually represent, we have added various tools to make working with expressions as easy as possible.
+
+import DiceKit
+
+//: Lets start with the negative-sided die.
+
+let negativeDie = d(-12)
+
+//: A die with negative sides has results in the range `-sides...-1`. We have two ways to represent an expression as a string.
```

Comment:
"A die with negative sides **by default** has results in the range `-sides...-1`."

Add "by default", and maybe reference that we have multiple ways to setup the roller? Maybe do that in the `Die` page, and just reference it here?